docs/examples/try_me_out/availability_search.ipynb

```diff
@@ -26,7 +26,18 @@
     },
     "tags": []
    },
-   "outputs": [],
+   "outputs": [
```

Comment:
Please clear these - this is going to put your file structure in the public domain :)